.circleci/config.yml

```diff
@@ -140,8 +140,7 @@ jobs:
       - checkout
       - attach_workspace:
           at: ~/circleci-champaign
-      - setup_remote_docker:
-          docker_layer_caching: true
```

Comment:
Whatt. I saw that this "isn't available on our plan". I hope it won't make builds much slower.